flake.nix

```diff
@@ -287,6 +289,21 @@
           '';
         };
 
+        nix-topology = {
+          title = "nix-topology";
+          baseUrl = "https://github.com/oddlama/nix-topology/blob/main";
+          attributePath = [ "flakeModule" ];
+          intro = ''
+            With nix-topology you can automatically generate infrastructure and network diagrams as SVGs
+            directly from your NixOS configurations, and get something similar to the diagram [here](https://github.com/oddlama/nix-topology#readme).
+            It defines a new global module system where you can specify what nodes and networks you have.
+            Most of the work is done by the included NixOS module which automatically collects all the
+            information from your hosts.
+
+            After including this flake-parts module, you can build your diagram by running `nix build .#topology.<current-system>.config.output`.
```

Comment:
I think it'd be ok to generate a [`perSystem.packages`](https://flake.parts/options/flake-parts#opt-perSystem.packages) package for this. That way users don't have to specify the `system`.
(I think you're referring to the cpu+os kind of system, right?)